### PR TITLE
Prometheus: (Experimental) Inject label matchers into queries (also change drone to fix ARM rpm build and Update Swagger)

### PR DESCRIPTION
**Pull request by kylebrandt**
**What is this feature?**
When label matchers are applied as an argument to the query, they are injected into the promql query before it is run.

Model (Edited with query inspector): 
image

Executed Query (As shown by query inspector):
image

TODO:
 - [x] Feature toggle (`promQLScope`)

**Why do we need this feature?**

[Add a description of the problem the feature is trying to solve.]

**Who is this feature for?**

[Add information on what kind of user the feature is for.]

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: public/app/plugins/datasource/prometheus/dataquery.cue [hunk 1]
```diff
@@ -45,6 +45,10 @@ composableKinds: DataQuery: {
 				// See https://github.com/grafana/grafana/issues/48081
 				intervalFactor?: number
 
+				scope?: {
+					matchers: string
+				}
+
```
Comment (itsmylife) [inline public/app/plugins/datasource/prometheus/dataquery.cue:51]: Can we get rid of this cue file? I don't think having it makes sense anymore. WDYT?

**Review by kminehart — APPROVED** [review]

File: pkg/tsdb/prometheus/models/scope.go [hunk 2]
```diff
@@ -0,0 +1,52 @@
+package models
+
+import (
+	"github.com/prometheus/prometheus/model/labels"
+	"github.com/prometheus/prometheus/promql/parser"
+)
+
+func ApplyQueryScope(rawExpr string, matchers []*labels.Matcher) (string, error) {
```
Comment (itsmylife) [inline pkg/tsdb/prometheus/models/scope.go:8]: I know it is experimental, but should we add some unit tests? At least for basic functionality?

File: pkg/tsdb/prometheus/models/query.go [hunk 3]
```diff
@@ -75,9 +78,14 @@ type Query struct {
 	RangeQuery    bool
 	ExemplarQuery bool
 	UtcOffsetSec  int64
+	Scope         Scope
```
Comment (itsmylife) [inline pkg/tsdb/prometheus/models/query.go:81]: Scope means many other things too and it is not obvious to someone who is not familiar with the feature. I think having a little comment line here will be quite useful

File: public/app/plugins/datasource/prometheus/dataquery.cue [hunk 4]
```diff
@@ -45,6 +45,10 @@ composableKinds: DataQuery: {
 				// See https://github.com/grafana/grafana/issues/48081
 				intervalFactor?: number
 
+				scope?: {
```
Comment (itsmylife) [inline public/app/plugins/datasource/prometheus/dataquery.cue:48]: Also we should have comment line here too. Explaining what scope is.

**Pull request merged**